apps/server/src/modules/common-cartridge/controller/dto/course.query.params.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, Matches } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+import { CommonCartridgeVersion } from '../../export/common-cartridge.enums';
+
+export class CourseQueryParams {
+	@IsString()
+	@Matches(Object.values(CommonCartridgeVersion).join('|'))
+	@ApiProperty({
+		description: 'The version of CC export',
+		required: true,
+		nullable: false,
```

Comment:
default from ApiProperty can be removed
otherwise ApiPropertyOptional

Reply:
done ✅